src/clarity/checkboxes/checkbox.spec.ts

```diff
@@ -134,6 +143,12 @@ describe("Checkbox", () => {
         expect(fixture.nativeElement.querySelector(".checkbox-inline")).not.toBeNull();
     });
 
+    it("supports indeterminate state", () => {
+        createTestComponent(IndetermiateCheckbox);
+        expect(fixture.nativeElement.querySelector(".checkbox-indeterminate")).not.toBeNull();
+        expect(fixture.componentInstance.checkboxInstance.indeterminate).toBe(true);
```

Comment:
@mathisscott - here is the test I wrote for indeterminate state. Good call on checking that its false - I'll add one for that.